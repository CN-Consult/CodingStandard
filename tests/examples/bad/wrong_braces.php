<?php
/**
 *
 * @file
 * @version 2.11
 * @copyright 2020 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */


for ($i=0; $i<10; $i++) {
	echo "Hello world!";
}

if (true) {
	echo "True is true";
} else {
	echo "True is false!?";
}

foreach ($a as $b) {
	echo "This is wrong also!";
}

while ($a=true) {
	echo "new lines are also for whiles!";
}