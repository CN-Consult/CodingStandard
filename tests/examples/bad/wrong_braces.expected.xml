<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/wrong_braces.php" errors="5" warnings="1" fixable="0">
        <error line="11" column="25" source="Squiz.ControlStructures.ControlSignature.CurlyBracketOpenOnNextLine" severity="5" fixable="0">Curly bracket of for should be on next line after for.</error>
        <warning line="15" column="1" source="Generic.CodeAnalysis.UnconditionalIfStatement.Found" severity="5" fixable="0">Avoid IF statements that are always true or false</warning>
        <error line="15" column="11" source="Squiz.ControlStructures.ControlSignature.CurlyBracketOpenOnNextLine" severity="5" fixable="0">Curly bracket of if should be on next line after if.</error>
        <error line="17" column="8" source="Squiz.ControlStructures.ControlSignature.CurlyBracketOpenOnNextLine" severity="5" fixable="0">Curly bracket of else should be on next line after else.</error>
        <error line="21" column="20" source="Squiz.ControlStructures.ControlSignature.CurlyBracketOpenOnNextLine" severity="5" fixable="0">Curly bracket of foreach should be on next line after foreach.</error>
        <error line="25" column="17" source="Squiz.ControlStructures.ControlSignature.CurlyBracketOpenOnNextLine" severity="5" fixable="0">Curly bracket of while should be on next line after while.</error>
    </file>
</phpcs>
